### ISSUE TITLE
Install terminal quit with output

### ISSUE THREAD

**Issue by Fretory** [microsoft/vscode-remote-release#1217]
`[06:36:46.087] remote-ssh@0.45.5
[06:36:46.087] win32 x64
[06:36:46.088] SSH Resolver called for "ssh-remote+alias", attempt 1
[06:36:46.088] SSH Resolver called for host: alias
[06:36:46.088] Setting up SSH remote "alias"
[06:36:46.105] Using commit id "f06011ac164ae4dc8e753a3fe7f9549844d15e35" and quality "stable" for server
[06:36:46.109] Testing ssh with ssh -V
[06:36:46.178] ssh exited with code: 0
[06:36:46.178] Got stderr from ssh: OpenSSH_for_Windows_7.6p1, LibreSSL 2.6.4
[06:36:46.179] Running script with connection command: ssh -o ClearAllForwardings=true alias bash
[06:36:46.180] Install and start server if needed
[06:36:46.441] > 
[06:36:46.441] Got some output, clearing connection timeout
[06:37:07.561] > ssh: connect to host xx.xx.xx.xx(IP) port 22: Connection timed out
> 过程试图写入的管道不存在。
> 
[06:37:07.928] "install" terminal command done
[06:37:07.928] Install terminal quit with output: 过程试图写入的管道不存在。
[06:37:07.928] Received install output: 过程试图写入的管道不存在。
[06:37:07.929] Failed to parse remote port from server output: 过程试图写入的管道不存在。
[06:37:07.929] 
[06:37:07.929] TELEMETRY: {"eventName":"resolver","properties":{"outcome":"failure","reason":"UnparsableOutput"},"measures":{"resolveAttempts":1,"retries":1}}
[06:37:07.929] ------`

I use aliyun,ubuntu 16.04.
vscode version:1.37.1

I have try all the ways that I can search,but it didn't work.

**Comment by atomicpapa** [comment]
I am also experiencing this issue as of today.  Everything was working fine and then just stopped.  Trying to reconnect resulted in similar output.

TELEMETRY: {"eventName":"resolver","properties":{"outcome":"failure","reason":"UnparsableOutput"},"measures":{"resolveAttempts":1,"retries":1}}

running VS Code Insiders v. 1.38.0 on Windows 10 and connecting to Ubuntu 18.04 server

**Comment by mykter** [comment]
Similarly here, it worked yesterday but not today. 
```
[10:55:38.803] Failed to parse remote port from server output: bc4af6ac-c789-403e-9664-8a2eb1f0778d==34613====ubuntu==
[10:55:38.803] 
[10:55:38.803] TELEMETRY: {"eventName":"resolver","properties":{"outcome":"failure","reason":"UnparsableOutput"},"measures":{"resolveAttempts":1,"retries":1}}

[10:58:45.298] "install" terminal command done
[10:58:45.298] Install terminal quit with output: 

```

Version: 1.38.0-insider (user setup)
Commit: 0f73473c08055054f317c1c94502f7f39fdbb164
Date: 2019-08-23T05:39:22.818Z
OS: Windows_NT x64 6.1.7601

**Comment by atomicpapa** [comment]
I fixed this by downgrading to 1.37.1. Sorry to be an issue with 1.38.0-insider

**Comment by roblourens** [comment]
This should be fixed. Please make sure the extension is up to date.

**Comment by umdstu** [comment]
For what it's worth, I'm still seeing this issue with:

VSCode 1.38.0 (user setup)
OS: Windows 10
Remote OS: CentOS 7
vscode-remote-extensionpack: 0.17

TLDR: Initial connection succeeds, subsequent connections fail...


It turns out iptables is blocking everything in my case, so I had to excplicitly add ingress/egress rules for port 43649. This enabled the initial connection to the remote server to succeed finally.

However, if you try to open a directory, a new vscode window is opened, re-connect attempt occurrs, and then fails with the same issue as above, even tho it's using the same port from what I can see in the logs.

**Comment by roblourens** [comment]
@umdstu you are describing a different issue, please open a new issue. I thought that only the ssh port would matter to the firewall because the server's port is being tunneled over ssh, but I might not understand your setup.

> So It would be helpful it something was logged other than the wget had an exit status of 0

Something is logged and setup will fail. If there's any followup please open a new issue with the log from the Remote-SSH output channel.

**Comment by coatsy** [comment]
> This should be fixed. Please make sure the extension is up to date.

This looks like it's regressed with the 2019.9.31312 release - rolling back to 2019.9.28260 seems to fix the problem
